easybuild/easyconfigs/w/wrapt/wrapt-1.15.0-foss-2023a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonBundle'
+
+name = 'wrapt'
+version = '1.15.0'
+
+homepage = 'https://pypi.org/project/wrapt/'
+description = """The aim of the wrapt module is to provide a transparent object
+proxy for Python, which can be used as the basis for the construction of
+function wrappers and decorator functions."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
With `SciPy-bundle` having moved toolchain from `foss` to `gfbf` we can do the same for `wrapt`.

Reply:
See also #18752 where I do the same in the 2022b toolchain generation.